src/javascript/app_2/App/Components/Form/digit_selector.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React     from 'react';
+
+const DigitSelector = ({
+    name,
+    onChange,
+    selected_digit,
+}) => {
+    const handleSelect = (item) => {
+        if (+item.target.getAttribute('data-value') !== selected_digit) {
+            onChange({ target: { name, value: +item.target.getAttribute('data-value') } });
+        }
+    };
+
+    return (
+        <div className='digit-selector center-text'>
```

Comment:
Why not add center text as a part of the `digit-selector` block?

Reply:
I think it's a common class, we use it in app, it's quite useful. do you think it's better to give center alignment to everything that needs it?

Reply:
Since the `center-text` belongs to the `digit-selector` block I think it's easier to read if it's within the actual `digit-selector` css block

Reply:
so we want to repeat styles instead of using common classes from now on?

Reply:
haha. that sounds like more refactoring than BEM, probably :) for now I've removed the center-text from here.